backend/service/chaos/experimentation/experimentstore/experiment_store.go

```diff
@@ -58,61 +50,66 @@ func New(cfg *any.Any, _ *zap.Logger, _ tally.Scope) (service.Service, error) {
 }
 
 func (fs *experimentStore) CreateExperiments(ctx context.Context, experiments []*experimentation.Experiment) error {
-	psql := sq.StatementBuilder.PlaceholderFormat(sq.Dollar)
-	builder := psql.Insert(experimentsTableName).Columns(experimentColumns...)
+	// This API call will eventually be broken into 2 separate calls:
+	// 1) creating the config
+	// 2) starting a new experiment with the config
+
+	// All experiments are created in a single transaction
+	tx, err := fs.db.Begin()
+	if err != nil {
+		return err
+	}
 
 	for _, experiment := range experiments {
-		marshaler := jsonpb.Marshaler{}
-		buf := &bytes.Buffer{}
-		err := marshaler.Marshal(buf, experiment.GetConfig())
+		// Step 1) create the config
+		configID := id.NewID()
+
+		configJson, err := marshalConfig(experiment)
 		if err != nil {
 			return err
 		}
-		s := buf.String()
-		builder = builder.Values(
-			id.NewID(),
-			s,
-		)
-	}
 
-	sql, args, err := builder.ToSql()
-	if err != nil {
-		return err
+		configSql := `INSERT INTO experiment_config (id, details) VALUES ($1, $2)`
+		_, err = fs.db.ExecContext(ctx, configSql, configID, configJson)
+		if err != nil {
+			return err
+		}
+
+		// Step 2) start a new experiment with the config
+		runSql := `
+			INSERT INTO experiment_run (
+                id, 
+		        experiment_config_id,
+		        start_time,
+			    end_time,                        
```

Comment:
Per our offline discussion that we had the other day - `scheduled_end_time` should be set to the same value `end_time` is set when a new experimentation_record is created.

Reply:
right but currently FI experiments aren't started with an end time. I'd like to get parity with the way things currently work and then add end_time to FI experiments in a followup change.

Reply:
https://jira.lyft.net/browse/RES-1552